packages/@sanity/desk-tool/src/pane/styles/contentStylesOverride.css

```diff
@@ -0,0 +1,13 @@
+@import 'part:@sanity/base/theme/variables-style';
+
+.root {
+  composes: root from '../../../../components/src/panes/styles/DefaultPane.css';
```

Comment:
This doesn't seem quite right? Should it have been imported as a part instead?

Reply:
Yes. It seems wrong. But exposing a part for this is also wrong. Im not comfortable exposing all CSS-parts without a documentation. I think all composing internally should be done like this, unless it is a documented part.

Reply:
But this includes it from outside of the package, and I'm 99% sure this will break in a standalone studio (where this code resides in `node_modules`). So either this has to be required from the `@sanity/components` module, or from a part.

Reply:
I understand. I will make a part of this.